src/renderer/widgets/WalletDetails/ui/ProxiesList.tsx

```diff
@@ -0,0 +1,133 @@
+import { groupBy } from 'lodash';
+import { useUnit } from 'effector-react';
+
+import { cnTw, copyToClipboard, toAddress } from '@shared/lib/utils';
+import { ChainTitle } from '@entities/chain';
+import { useI18n } from '@app/providers';
+import { Accordion, DropdownIconButton, FootnoteText, HelpText, IconButton } from '@shared/ui';
+import type { Chain, ID, ProxyAccount } from '@shared/core';
+import { EmptyProxyList, ProxyAccount as ProxyAccountComponent, proxyModel } from '@entities/proxy';
+import { chainsService } from '@entities/network';
+import { AssetBalance } from '@entities/asset';
+import { DropdownIconButtonOption } from '@shared/ui/Dropdowns/common/types';
+import { ExplorersPopover } from '@entities/wallet';
+import { walletProviderModel } from '../model/wallet-provider-model';
+
+type Props = {
+  walletId: ID;
+  canCreateProxy?: boolean;
+  className?: string;
+};
+
+export const ProxiesList = ({ walletId, className, canCreateProxy = true }: Props) => {
+  const { t } = useI18n();
+
+  const proxyAccounts = useUnit(walletProviderModel.$proxyAccounts);
+  const proxyChainGroups = useUnit(proxyModel.$proxyChainGroupStore)[walletId];
+
+  if (!proxyAccounts || !proxyAccounts.length) return <EmptyProxyList className={className} />;
+
+  const proxiesByChain = groupBy(proxyAccounts, 'chainId');
+
+  const forgetProxyAction: DropdownIconButtonOption = {
+    icon: 'forget',
+    title: t('walletDetails.common.removeProxyAction'),
+    onClick: () => {},
+  };
+  const openInfoAction: DropdownIconButtonOption = {
+    icon: 'info',
+    title: t('walletDetails.common.openInfoAction'),
+    onClick: () => {},
+  };
+
+  const getProxyActionButton = (proxyAccount: ProxyAccount, chain: Chain) => {
+    const proxiedAddress = toAddress(proxyAccount.proxiedAccountId, { prefix: chain.addressPrefix });
+
+    return (
+      <DropdownIconButton name="more" className="ml-2">
+        <DropdownIconButton.Items>
+          <DropdownIconButton.Item>
+            <ExplorersPopover
+              address={proxyAccount.accountId}
+              explorers={chain.explorers || []}
+              addressPrefix={chain.addressPrefix}
+              className="-mt-10 -mr-1"
```

Comment:
I don't like that for strange situations with negative margins we are forced to add `className` though we easily lived without it for a month or so
Could you provide a screenshot so we could discuss details and come up with another solution?